polygeist/lib/Dialect/Polygeist/Transforms/ArgumentPromotion.cpp

```diff
@@ -347,8 +325,13 @@ void Candidate::modifyCallee() {
 
   funcOp.setType(newFuncType);
   funcOp.setAllArgAttrs(newArgAttrs);
-  if (isLinkonceODR(callableOp))
-    privatize(callableOp);
+
+  // Change the linkage from linkonce_odr to internal.
```

Comment:
The comment says we set the linkage to `internal`, but the changes to the test actually replace `internal` with `private`. Should we use `private` in this comment here to keep it consistent with the actual name?